templates/cli/lib/type-generation/languages/java.js.twig

```diff
@@ -44,7 +44,7 @@ class Java extends LanguageMeta {
 import java.util.*;
 <% for (const attribute of collection.attributes) { -%>
 <% if (attribute.type === 'relationship') { -%>
-import <%- toPascalCase(attribute.relatedCollection) %>;
+import io.appwrite.models.<%- toPascalCase(attribute.relatedCollection) %>;
```

Comment:
are we sure we hardcode this value, (io.appwrite) feels like should be coming from some config like sdk namespace or something. Don't we have this somewhere else as well? is it hardcoded there too?

Reply:
Let's note it down for later changes.